scripts/zones/Mount_Zhayolm/npcs/HomePoint#1.lua

```diff
@@ -24,6 +24,7 @@ end;
 function onTrigger(player,npc)
 
 	homepointMenu( player, 0x21fc, 90);
+    printf("CSID: %u",csid);
```

Comment:
fixed. 
and for the error, in npc_list.sql i for got i had stop on errors turned off. and i did not notice any error's. oops sorry. working on fix and rechecking as I type this.